dnf+

```diff
@@ -0,0 +1,19 @@
+---
+tags: [ packaging ]
+---
+# To install a package:
+dnf install <package>
+
+# To find packages matching <phrase>:
+dnf search <phrase>
```

Comment:
Multiple fixes available: please chose one of them.

```suggestion
# Find packages matching a specific phrase:
dnf search "<phrase>"
```

Almost the same as above.

```suggestion
# Find packages matching one of specific phrases:
dnf search "<phrase1 phrase2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple phrases at once for `search` command then u can use this fix.